public/css/PMR.css

```diff
@@ -1,24 +1,42 @@
- /*subject to change*/
-input[type=text], select {  
-    font-family: "Lucida Console"; 
-    font-size: 15px;
-    width: 50%;
-    padding: 15px 30px 15px; /*top, right, bottom, left */
-    margin: 10px 0px;
-    display: inline-block;
-    border: 1px solid #ccc;
-    border-radius: 4px;
-    box-sizing: border-box;
+.dropdown{	//dropdown button
+    font-family: "Lucida Console";
+    font-size: 10px;
+    margin: 10px 0;
+    padding: 15px 30px;
 }
 
-.chart-container {       /*Update colour scheme*/
-  font: 10px "Lucida Console";
-  background-image: linear-gradient(180deg, #ffffff, #99ccff); /*gradient?/ different for each*/
-  border-radius: 0.5em;
-  margin-top: 2em;
-  height: 500px;
+.chart-container {	
+    background-color: #eceff1;
+    border-radius: 0.5em;
+    font: 10px "Lucida Console";
+    height: 500px;
+    margin-top: 2em;
 }
-
 .chart {
-  height: 100%;
-}
\ No newline at end of file
+    height: 100%;
+}
+body {
+    background-color: #eceff1;
+}
+.navbar-default {	
+    background-color: #64b5f6;
+}
+.navbar-default .navbar-brand {	//Navbar brand on left of navbar
+    color: #ffffff;
+  	font-family: "Trebuchet MS";
+    font-size: 25px;
+}
+.navbar-default .navbar-brand:hover {	//when navbar brand is hovered
+    color: #263238;
+}
+.navbar-default .navbar-nav > li > a {	//admin button on right of navbar
+    color: #263238;
```

Comment:
I'm not sure if this colour really works. It seems to dark to go against the blue background. I'd suggest making it white unless we wanted to keep it as a dark grey.